mirar/processors/sources/source_detector.py

```diff
@@ -230,6 +234,55 @@ def _apply_to_images(
                 diff_scorr_path=scorr_image_path,
             )
 
+            logger.debug(f"Found {len(srcs_table)} candidates in positive image")
+            if self.detect_in_negative:
```

Comment:
what does detect_in_negative do? Can you think of a slightly more explicit name? 

From the name I'd guess it just uses the negative image instead of positive, but from the code I think it actually does both rather than positive only.

Reply:
Changed to `detect_negative_sources` and added docstrings